index.html

```diff
@@ -120,36 +99,64 @@ <h4 class="experience-heading">Skills</h4>
             </ul>
-            <button type="submit" class="project-button">See Project</button>
+            <button type="submit" class="btn btn-primary project-button see-project" data-project="0">See Project</button>
         </div>
         <br>
         <br>
     </section>
+    <div class="project-image-2"></div>
+    <div class="modal" id="myModal">
+        <div class="modal-dialog modal-xl">
+            <div class="modal-content">
+                <div class="modal-body">
+                    <div class="flex-hd">
+                        <h5 class="modal-title">Multi-Post stories</h5>
+                        <button type="button" class="btn-close btn-close-white" data-bs-dismiss="modal"></button>
+                    </div>
+                    <div class="flex-bd">
+                        <img src="images/pop-up-image.png" alt="pop up image" class="pop-image">
+                        <p class="pop-up-paragraph">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.</p>
+                        <ul class="project-list">
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">Bootstrap</a></li>
+                            <li><a href="#">Ruby</a></li>
+                        </ul>
+                        <div class="see-btn">
+                            <div class="btn-style">
+                                <a class="see-live" id="seelive">See Live <img src="images/seelive.png" alt=""> </a>
+                            </div>
+
+                            <div class="btn-style">
+                                <a class="see-source" id="seeSource">See Source  <img src="images/Vector-1.png" alt=""> </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+    </div>
+
     <section class="work-section">
-        <div class="project-holder-2"></div>
-        <div class="project-image-2"></div>
         <div class="project-section">
+            <div class="project-holder-2 "></div>
             <h2>Multi-Post Stories</h2>
             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
```

Comment:
Hi @ababaug,

Good job creating the projects array (pop-up-windows.js file)👍 , however, you need to render the projects section using the information in the array you created, don't hardcode them in this file (index.html), kindly implement it just as it is stated in the[ project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20update%20the%20main%20page%20so%20that%20the%20projects%20section%20is%20created%20dynamically%20using%20the%20information%20stored%20in%20that%20JavaScript%20object)